.gitignore

```diff
@@ -36,6 +36,7 @@ examples/pkcs7/pkcs7
 pkcs7tpmsigned.p7s
```

Comment:
Please add 
`pkcs7tpmsignedex.p7s`

Reply:
That file is created when running `./examples/pkcs7/pkcs7` and should not be tracked in repo.